javascript/index.html

```diff
@@ -76,7 +76,7 @@
 } else {
   // This code could also be modified to ask for access to the user's genomics data too
   // You just need one additional scope: 'https://www.googleapis.com/auth/genomics'
-  // And an additional library: {name: 'genomics', version: 'v1beta'}
+  // And an additional library: {name: 'genomics', version: 'v1'}
```

Comment:
This change will not work. The googlegenomics.jquery.js that it depends on will send requests to the wrong endpoint:

  * www.googleapis.com

instead of:

  * genomics.googleapis.com

I propose that when I have fixed things in the api-client-javascript (which I am currently working on), I will move this example to that repository.